health-services/project-factory/src/server/api/coreApis.ts

```
@@ -0,0 +1,259 @@
+ * 
+ * @remarks
+ * This function queries the boundary relationship API to retrieve hierarchy data
+ * based on the specified hierarchy type and inclusion of child or parent entities.
+ * 
+ * @example
+ * const response = await searchBoundaryRelationshipData("mz", "ADMIN", true, false);
+ */
+const searchBoundaryRelationshipData = async (
+  tenantId: string,
+  hierarchyType: string,
+  includeChildren: boolean = true,
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)